packages/fluentui/react-northstar/src/components/Carousel/CarouselPaddlesContainer.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {
+  useFluentContext,
+  useTelemetry,
+  useStyles,
+  useAccessibility,
+  getElementType,
+  useUnhandledProps,
+  ComponentWithAs,
+} from '@fluentui/react-bindings';
+import {
+  commonPropTypes,
+  UIComponentProps,
+  ContentComponentProps,
+  ChildrenComponentProps,
+  childrenExist,
+} from '../../utils';
+import { FluentComponentStaticProps } from '../../types';
+import { Accessibility } from '@fluentui/accessibility';
+
+export interface CarouselPaddlesContainerProps extends UIComponentProps, ContentComponentProps, ChildrenComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<never>;
+}
+
+export type CarouselPaddlesContainerStylesProps = never;
+export const carouselPaddlesContainerClassName = 'ui-carrouselpaddles_container';
+
+/**
+ * A CarouselPaddlesContainer is a container for the Carousel Paddles.
+ */
+export const CarouselPaddlesContainer: ComponentWithAs<'span', CarouselPaddlesContainerProps> &
```

Comment:
From what I can see, this is not actually a span ? 

```suggestion
export const CarouselPaddlesContainer: ComponentWithAs<'div', CarouselPaddlesContainerProps> &
```